docs-source/content/samples/simple/azure-kubernetes-service/_index.md

```diff
@@ -118,24 +104,34 @@ Successful output will look like the following:
 
 ##### Oracle Container Registry
 
-You will need an Oracle account.  Follow the steps at [GET IMAGES](/weblogic-kubernetes-operator/quickstart/get-images/), accepting the license agreement for WebLogic Server in the process.  Make note of your Oracle Account password and email.  This sample pertains to 12.2.1.3, but other
-versions may work as well.
+You will need an Oracle account. The following steps will direct you to accept the license agreement for WebLogic Server.  Make note of your Oracle Account password and email.  This sample pertains to 12.2.1.3, but other versions may work as well.
+
+1. Obtain the WebLogic Server image from the [Oracle Container Registry](https://container-registry.oracle.com/).
+
+   a. First time users, [follow these directions](/weblogic-kubernetes-operator/userguide/managing-domains/domain-in-image/base-images/#obtaining-standard-images-from-the-oracle-container-registry).
```

Comment:
Not sure that your hyperlink to https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-in-image/base-images/#obtaining-standard-images-from-the-oracle-container-registry is working. Please check.

Reply:
It worked for me in Chrome.

Reply:
After merge to master, the above link does work correctly on Chrome.